### ISSUE TITLE
Feature request. Bowpad could use a way of turning off backup file usage.

### ISSUE THREAD

**Issue by gmcode** [stefankueng/BowPad#122]
Feature request:
I'd rather not use the process where by bowpad allows me to quit bowpad with an unsaved file with it saving the partial edit to a backup place and reloading that instead of the original file when it exists.

I find I am often accidentally leaving half edited files, then editing the original again from another VM etc. with the original usually being what I want to keep at that time as I explicitly saved it. But then when I come back to the first environment Bowpad now has a dilemma and I can then end up overwriting my original file with the backup accidentally.

I'd rather just turn off the backup feature so that I can't get into this situation in the first place. It'd be great to be off by default. I appreciate the feature is useful sometimes and that other people will like the feature. But it'd prefer to explicitly enable such a feature.

Apologies if a feature already exists to turn the backups off. If it does, could you please advise me of it.
Thanks

**Comment by wmjordan** [comment]
Have you tried to turn off the session auto-load?

**Comment by gmcode** [comment]
Hi thanks for the suggestion, but I don't think turning off auto-load is what I want to do.
auto load is related to automatically loading the last files I edited. When I have saved those files, I still want to auto load them. I just don't want to autoload the "backup" files that I haven't saved.

What I want is that if bowpad sees I have not saved a file, it just forces me to save it or not save it before exiting. I want an option so bowpad won't on exit automatically save my unsaved file to a backup location and autoload that next time.

I hope that makes sense.
